core/htmldataprocessor.js

```diff
@@ -107,7 +107,7 @@
 			// structure.
 			var el = editor.document.createElement( fixBin );
 			// Add fake character to workaround IE comments bug. (https://dev.ckeditor.com/ticket/3801)
-			el.setHtml( 'a' + data );
+			el.setHtml( CKEDITOR.tools.htmlSafeByReview('a' + data, 'The data variable is heavily sanitized before being passed here as seen in the previous lines.'));
```

Comment:
It's probably safe but we're not sure, so we should add a legacy conversion.

Reply:
Will be legacy converted.